euphoria-service/src/main/java/edu/cooper/ece366/euphoria/Authentication.java

```diff
@@ -4,6 +4,8 @@
 
 @AutoMatter
 public interface Authentication {
+    Integer Id();
```

Comment:
Just to document for ourselves

```suggestion
    Integer Id();  // Either userId or companyId
```